app/src/main/java/com/humberto/tasky/agenda/presentation/AgendaItemType.kt

```diff
@@ -0,0 +1,7 @@
+package com.humberto.tasky.agenda.presentation
+
+sealed class AgendaItemType {
+    data object Task : AgendaItemType()
+    data object Event : AgendaItemType()
+    data object Reminder : AgendaItemType()
+}
```

Comment:
If there's nothing individual about these different values, could it make more sense to make this an enum class instead? This comes with some useful extensions like being able to get all types as a list: `AgendaItemType.entries`